.github/workflows/label-all.yml

```diff
@@ -0,0 +1,14 @@
+name: "Issue and PR Labeler"
+on:
+  pull_request_target:
+    types: [opened]
+  issues:
+    types: [opened, reopened]
+jobs:
+  label-all-on-open:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: andymckay/labeler@main
```

Comment:
Can we use a specific release instead of pointing to `main`?

Reply:
sure!